notebook/i18n/nbjs.json

```diff
@@ -2,6 +2,7 @@
 	"domain": "nbjs",
 	"supported_languages": [
 		"fr-FR",
+		"ja_JP",

```

Comment:
What browser are you using? This list should probably be consistent in using `-` or `_`, and if different browsers do different things, we may need to work around that.

Reply:
Google Chrome. 

![image](https://user-images.githubusercontent.com/10111/56976542-2920ae80-6bae-11e9-851f-4bd05f41531a.png)

The name of UNIX locale on japanese/japan is `ja_JP`. But the browser send `ja`.